src/core/lv_obj_draw.c

```diff
@@ -27,6 +27,8 @@
  *  STATIC PROTOTYPES
  **********************/
 
+static inline lv_opa_t try_get_layer_opa(const lv_obj_t * obj, lv_part_t part, const lv_draw_dsc_base_t * base_dsc);
```

Comment:
```suggestion
static inline lv_opa_t get_layer_opa(const lv_obj_t * obj, lv_part_t part, const lv_draw_dsc_base_t * base_dsc);
```

I suggest removing "try" as it will always get it somehow.